include/cantera/oneD/MultiNewton.h

```diff
@@ -45,11 +45,61 @@ class MultiNewton
                      const OneDim& r, int loglevel);
 
     /**
-     * On entry, step0 must contain an undamped Newton step for the solution x0.
-     * This method attempts to find a damping coefficient such that the next
-     * undamped step would have a norm smaller than that of step0. If
-     * successful, the new solution after taking the damped step is returned in
-     * x1, and the undamped step at x1 is returned in step1.
+     * Performs a damped Newton step to solve the system of nonlinear equations.
+     *
+     * On entry, `step0` must contain an undamped Newton step for the solution `x0`.
+     * This method attempts to find a damping coefficient `alpha_k` such that the next
+     * undamped step would have a norm smaller than that of `step0`. If successful,
+     * the new solution after taking the damped step is returned in `x1`, and the
+     * undamped step at `x1` is returned in `step1`.
+     *
+     * This uses the method outlined in @cite kee2003.
```

Comment:
In Doxygen, the `@cite kee2003` just gets turned into `[17]`, so this should read:
```suggestion
     * This uses the method outlined in Kee et al. @cite kee2003.
```
even though this looks slightly redundant when reading the raw text.